installer/openSUSE/releases

```diff
@@ -0,0 +1 @@
+tumbleweed
```

Comment:
it's not supported at the moment, I'll evaluate if dropping the leap part for now.